docker-entrypoint-funcs.sh

```diff
@@ -1026,5 +1027,5 @@ _edbdocker_watch_cert() {
     "=============================================================== "
   )
   edbdocker_log_at_level "info" "${msg[@]}"
-  edbdocker_log_at_level "info" "$(cat ${cert_path})"
+  edbdocker_log_at_level "info" "$(cat "${cert_path}")"
```

Comment:
This one too - nested quotes?

Reply:
Nested quotes in command substitution are not just valid, but also recommended.  This is because everything inside `$()` is executed as-is in a subshell.

Reply:
Here's the [relevant bit](https://pubs.opengroup.org/onlinepubs/009604499/utilities/xcu_chap02.html#tag_02_02_03) from the POSIX standard, if you're curious:

> Within the string of characters from an enclosed "${" to the matching '}', an even number of unescaped double-quotes or single-quotes, if any, shall occur.

Reply:
Thanks!